documentation/staging/content/release-notes.md

```diff
@@ -47,7 +47,11 @@ draft: false
 
 #### Operator 3.3.2
 
+* Support for the networking changes included with Istio 1.10 ([#2538](https://github.com/oracle/weblogic-kubernetes-operator/pull/2538)).
+* Support for accessing the WebLogic console through `kubectl port-forward` ([#2520](https://github.com/oracle/weblogic-kubernetes-operator/pull/2520)).
```

Comment:
the WebLogic console -> are you referring to the WebLogic Server Administration Console? (IIRC, the doc refers to the Administration Console, the Remote Console, and WLST). Not sure if you want to be more specific, but I wouldn't know what you are referring to by "WebLogic console." If you want a minimal improvement, maybe call it the Administration Console or WebLogic consoles? (Just a suggestion; I'm sure it's ok to leave it as is, also.)